stock_mts_mto_mrp_rule/tests/test_stock_move.py

```diff
@@ -0,0 +1,5 @@
+from odoo.addons.stock.tests.common import TestStockCommon
+
+
+class TestStockMove(TestStockCommon):
+    pass
```

Comment:
@geomer198 Do you plan to update this ? :smiley:

Reply:
@rousseldenis Of Course!